ladder/teams/api.yaml

```diff
@@ -1,4 +1,3 @@
 members:
 - aanm
 - nathanjsweet
-- vadorovsky
```

Comment:
fyi @vadorovsky if you'd like to review some PRs, please reach out.

Reply:
I'm fine with being removed, I haven't been active here in years.